ArduCopter/Parameters.cpp

```diff
@@ -492,6 +492,16 @@ const AP_Param::Info Copter::var_info[] = {
     // @Group: IM_
     // @Path: ../libraries/AC_InputManager/AC_InputManager_Heli.cpp
     GOBJECT(input_manager, "IM_", AC_InputManager_Heli),
+
+    #if MODE_AUTOROTATE_ENABLED == ENABLED
+        // @Group: AROT_
+        // @Path: ../libraries/AC_Autorotation/AC_Autorotation.cpp
+        GOBJECTPTR(arot, "AROT_", AC_Autorotation),
+
+        // @Group: SPDHGT_
+        // @Path: ../libraries/AP_SpdHgtControl/AP_SpdHgtControl_Heli.cpp
+        GOBJECTPTR(helispdhgtctrl, "SPDHGT_",   AP_SpdHgtControl_Heli),
+    #endif
 #endif
```

Comment:
Per last night's Dev call, these will need to be moved into the G2 parameter grouping below.  Looks like the syntax is slightly different.

Reply:
Having a spot of trouble with this.  I have moved the parameters over to the G2 group and the code builds.  However, I cant see the parameters in MP.  I have pushed a commit with my changes.  Would appreciate it anyone could have a look over it to see if they can spot what I am doing wrong.

Reply:
This is now done.